test/unit/proxy_test.rb

```diff
@@ -148,6 +148,16 @@ def test_endpoint_validation
 
       service.deployment_option = 'plugin_ruby'
```

Comment:
Oh another good cleanup to do later.

I think there is no more `plugin_*`, another cleanup to add to our list :)

Reply:
@hallelujah Okay will add a new ticket later for this issue to work on , For now I will fix the save! issue.